server/src/main/java/com/unosquare/carmigo/controller/UserControllerHelper.java

```diff
@@ -0,0 +1,66 @@
+package com.unosquare.carmigo.controller;
+
+import com.github.fge.jsonpatch.JsonPatch;
+import com.unosquare.carmigo.dto.CreateDriverDTO;
+import com.unosquare.carmigo.dto.GrabDriverDTO;
+import com.unosquare.carmigo.dto.GrabPassengerDTO;
+import com.unosquare.carmigo.dto.GrabPlatformUserDTO;
+import com.unosquare.carmigo.model.request.CreateDriverViewModel;
+import com.unosquare.carmigo.model.response.DriverViewModel;
+import com.unosquare.carmigo.model.response.PassengerViewModel;
+import com.unosquare.carmigo.model.response.PlatformUserViewModel;
+import com.unosquare.carmigo.service.UserService;
+import lombok.RequiredArgsConstructor;
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+class UserControllerHelper {
```

Comment:
I am curious why you think we need a controller helper?   Would this not be part of the service?

Reply:
The idea is to avoid code repetition that would happen in UserController.java and AdminUserController.java e.g.
* map an incoming View Model to DTO
* and then map DTO to an outgoing View Model

Reply:
Removed controller helper